st3/sublime_lib/view_utils.py

```diff
@@ -13,7 +13,7 @@
 
 __all__ = [
     'LineEnding', 'new_view', 'close_view',
-    'clone_view', 'temporarily_scratch_unsaved_views',
```

Comment:
Exporting an underscored name in `__all__` kind of defeats the purpose of underscoring it in the first place, doesn't it? Those should be mutually exclusive. 

The functions are for internal use only and we'll never use a star import in our own code, I presume, making this redundant.

Reply:
I know it doesn't do anything, but for some reason I thought that mypy used it as a hint to check imports. Apparently it doesn't, so I've removed them from `__all__`.